descriptors/ModuleDescriptor-template.json

```diff
@@ -20,6 +20,14 @@
           "methods": [ "POST" ],
           "pathPattern": "/refresh",
           "permissionsRequired": []
+        },
+        {
+          "methods": [ "POST" ],
+          "pathPattern": "/_/tenant"
+        },
+        {
+          "methods": [ "GET", "DELETE" ],
+          "pathPattern": "/_/tenant/{id}"
         }
```

Comment:
These endpoints *must* be part of the _tenant version 2 interface. So authtoken provides two interfaces: `authtoken` and `_tenant.`